src/libstd/process.rs

```diff
@@ -1444,7 +1443,7 @@ pub fn id() -> u32 {
 #[unstable(feature = "termination_trait_lib", issue = "43301")]
 #[rustc_on_unimplemented(
   message="`main` has invalid return type `{Self}`",
-  label="`main` can only return types that implement {Termination}")]
+  label="`main` can only return types that implement `{Termination}`")]
```

Comment:
Can you add a test that actually emits this message? Or find one and extend it with the entire message so this change is reflected in a test?